src/main/resources/db/migration/changes/UCSBDates.json

```diff
@@ -27,7 +27,7 @@
                       "autoIncrement": true,
                       "constraints": {
                         "primaryKey": true,
-                        "primaryKeyName": "CONSTRAINT_5"
+                        "primaryKeyName": "CONSTRAINT_7"
```

Comment:
Might need to pull from main, this is a fix they implemented a few days ago so shouldn't change it

Could also just unstage it